cli/js/buffer.ts

```diff
@@ -157,31 +157,27 @@ export class Buffer implements Reader, ReaderSync, Writer, WriterSync {
 
   async readFrom(r: Reader): Promise<number> {
     let n = 0;
+    const buf = new Uint8Array(32 * 1024);
```

Comment:
calling `this.#grow` without knowing the actual amount that needs to grow is causing troubles, went with the simpler and less error-prone code, which is calling `writeSync` which calls `this.#grow` with the size of the read buffer.

In a few minutes, I'll push an alternative reading directly to the buffer (which is not as clean). If someone else wants to try I'm more than happy.

Reply:
Posted alternative implementation in #6570